adafruit_wiznet5k/adafruit_wiznet5k_dhcp.py

```diff
@@ -132,6 +132,11 @@ def __init__(self, eth, mac_address, timeout=1, timeout_response=1, debug=False)
         self._t1 = 0
         self._t2 = 0
 
+        # Host name
+        mac_string = "".join("{:02X}".format(o) for o in mac_address)
+        self._hostname = bytes((hostname or "WIZnet{}")
```

Comment:
Noting before I test, `hostname` is set to `none` in L191 `set_dhcp`, might fail here if a hostname is not predefined in`__init__`